web/src/settings_org.ts

```diff
@@ -91,39 +106,50 @@ export function maybe_disable_widgets() {
-    options.common_policy_values = settings_components.get_sorted_options_list(
-        settings_config.common_policy_values,
-    );
-    options.wildcard_mention_policy_values = settings_components.get_sorted_options_list(
-        settings_config.wildcard_mention_policy_values,
-    );
-    options.common_message_policy_values = settings_components.get_sorted_options_list(
-        settings_config.common_message_policy_values,
-    );
-    options.invite_to_realm_policy_values = settings_components.get_sorted_options_list(
-        settings_config.email_invite_to_realm_policy_values,
-    );
-    options.edit_topic_policy_values = settings_components.get_sorted_options_list(
-        settings_config.edit_topic_policy_values,
-    );
-    options.move_messages_between_streams_policy_values =
-        settings_components.get_sorted_options_list(
+type OrganizationSettingsOptions = {
+    common_policy_values: SettingOptionValueWithKey[];
+    wildcard_mention_policy_values: SettingOptionValueWithKey[];
+    common_message_policy_values: SettingOptionValueWithKey[];
+    invite_to_realm_policy_values: SettingOptionValueWithKey[];
+    edit_topic_policy_values: SettingOptionValueWithKey[];
+    move_messages_between_streams_policy_values: SettingOptionValueWithKey[];
+};
+
+export function get_organization_settings_options(): OrganizationSettingsOptions {
+    return {
+        common_policy_values: settings_components.get_sorted_options_list(
+            settings_config.common_policy_values,
+        ),
+        wildcard_mention_policy_values: settings_components.get_sorted_options_list(
+            settings_config.wildcard_mention_policy_values,
+        ),
+        common_message_policy_values: settings_components.get_sorted_options_list(
+            settings_config.common_message_policy_values,
+        ),
+        invite_to_realm_policy_values: settings_components.get_sorted_options_list(
+            settings_config.email_invite_to_realm_policy_values,
+        ),
+        edit_topic_policy_values: settings_components.get_sorted_options_list(
+            settings_config.edit_topic_policy_values,
+        ),
+        move_messages_between_streams_policy_values: settings_components.get_sorted_options_list(
             settings_config.move_messages_between_streams_policy_values,
-        );
-    return options;
+        ),
+    };
 }
 
-export function get_org_type_dropdown_options() {
+type DefinedOrgTypeValues = typeof settings_config.defined_org_type_values;
+type AllOrgTypeValues = typeof settings_config.all_org_type_values;
+
+export function get_org_type_dropdown_options(): DefinedOrgTypeValues | AllOrgTypeValues {
     const current_org_type = realm.realm_org_type;
     if (current_org_type !== 0) {
         return settings_config.defined_org_type_values;
     }
     return settings_config.all_org_type_values;
 }
 
-const simple_dropdown_properties = [
+const simple_dropdown_properties: (keyof SimpleDropdownRealmSettings)[] = [
```

Comment:
It looks like`"realm_create_private_stream_policy"` just isn't being used, since it's not in the DOM the way the other keys are. I think we can remove it.

Reply:
Looks like it was replaced with `realm_can_create_private_channel_group` in e19524ba92eca6365504fd4016ff6c6bba4d7be5